content/docs/1.6.0/best-practices.md

```diff
@@ -52,12 +52,12 @@ The following Linux OS distributions and versions have been verified during the
 | No. | OS           | Versions
 |-----|--------------| --------
 | 1.  | Ubuntu       | 22.04
-| 2.  | SLES         | 15 SP4
-| 3.  | SLE Micro    | 5.4
-| 4.  | RHEL         | 9.1
-| 5.  | Oracle Linux | 9.1
-| 6.  | Rocky Linux  | 9.2
-| 7.  | Talos Linux  | 1.6.0
+| 2.  | SLES         | 15 SP5
```

Comment:
Remember, we also need to update the actively supported branches (1.5.x and 1.4.x) for SLES and SLE Micro, I believe we already updated the pipelines but the doc was not.

Reply:
Updated the sles version, https://github.com/longhorn/website/pull/849. For SLE Micro, will wait for the upgrade testing completion of 1.6.0. We don't run daily test and upgrade test on SLE on daily basis.